src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/AdditionalCondition.kt

```diff
@@ -20,19 +21,18 @@ data class AdditionalCondition(
   @Id
   @GeneratedValue(strategy = IDENTITY)
   @NotNull
-  val id: Long = -1,
+  val id: Long? = null,
```

Comment:
I've just copied the `-1` solution because it means that the property is not nullable which reflects the real model... What was the reason for the switch to nullable?

Reply:
Reverted to the non-nullable field, thanks!